versioned_docs/version-v1.3/troubleshooting/host.md

```diff
@@ -0,0 +1,139 @@
+
+```
+- apiVersion: longhorn.io/v1beta2
+  kind: VolumeAttachment
+  spec:
+    attachmentTickets:
+      csi-b5097155cddde50b4683b0e659923e379cbfc3873b5b2ee776deb3874102e9bf:
+        id: csi-b5097155cddde50b4683b0e659923e379cbfc3873b5b2ee776deb3874102e9bf
+        nodeID: node-name
+...
+    volume: pvc-3c6403cd-f1cd-4b84-9b46-162f746b9667
+```
+
+:::note
+
+Manually attaching a volume to the node is not recommended.
+
+:::
+
+#### Workaround 1: Set `Detach Manually Attached Volumes When Cordoned` to `True`
+
+The Longhorn setting [Detach Manually Attached Volumes When Cordoned](https://longhorn.io/docs/1.6.0/references/settings/#detach-manually-attached-volumes-when-cordoned) blocks node draining when there are volumes manually attached to the node.
+
+The default value of this setting depends on the embedded Longhorn version:
+
+| Harvester version | Embedded Longhorn version | Default value |
+| --- | --- | --- |
+| v1.3.1 | v1.6.0 | `true` |
+| v1.4.0 | v1.7.0 | `false` |
+
+Set this option to `true` from the [embedded Longhorn UI](./harvester.md#access-embedded-rancher-and-longhorn-dashboards).
+
+#### Workaround 2: Manually Detach the Volume
+
+Detach the volume using the [embedded Longhorn UI](./harvester.md#access-embedded-rancher-and-longhorn-dashboards).
+
+![detached-volume.png](/img/v1.3/troubleshooting/detached-volume.png)
+
+Once the volume is detached, you can successfully enable Maintenance Mode on the node.
+
+![node-enter-maintenance-mode.png](/img/v1.3/troubleshooting/node-enter-maintenance-mode.png)
+
+### Single-Replica Volumes
+
+Harvester allows you to create custom StorageClasses that describe how Longhorn must provision volumes. If necessary, you can create a StorageClass with the [Number of Replicas](../advanced/storageclass.md#number-of-replicas) parameter set to `1`.
+
+When a volume is created using such a StorageClass and is attached to a node using the CSI driver or other methods, the lone replica stays on that node even after the volume is detached.
+
+You can check this using the CRD object `Volume`.
+
+```
+- apiVersion: longhorn.io/v1beta2
+  kind: Volume
+...
+  spec:
+...
+    numberOfReplicas: 1  // the replica number
+...
+  status:
+...
+    ownerID: nodeName
+...
+    state: attached
+```
+
+#### Workaround: Set `Node Drain Policy`
+
+The Longhorn [Node Drain Policy](https://longhorn.io/docs/1.6.0/references/settings/#node-drain-policy) is set to `block-if-contains-last-replica` by default. This option forces Longhorn to block node draining when the node contains the last healthy replica of a volume.
+
+To address the issue, change the value to `allow-if-replica-is-stopped` using the [embedded Longhorn UI](./harvester.md#access-embedded-rancher-and-longhorn-dashboards).
+
+:::info important
+
+If you plan to remove the node after Maintenance Mode is enabled, back up single-replica volumes or redeploy the related workloads to other nodes in advance so that the volumes are scheduled to other nodes.
```

Comment:
```suggestion
If you plan to remove the node after `Maintenance Mode` is enabled, back up single-replica volumes or redeploy the related workloads to other nodes in advance so that the volumes are scheduled to other nodes.
```